src/ensembl/src/content/app/entity-viewer/gene-view/components/transcripts-table/main-content/MainContent.scss

```diff
@@ -0,0 +1,5 @@
+.mainContent {
+  & > div > div {
+    grid-template-columns: 2fr 1.5fr 1.5fr 1fr 1.5fr 1fr;
```

Comment:
As in other places of where we use CSS grid for this table, we probably want actual pixels rather than fractions of available width here